src/webgpu/shader/execution/expression/call/builtin/pack2x16snorm.spec.ts

```diff
@@ -17,4 +31,28 @@ g.test('pack')
 @const fn pack2x16snorm(e: vec2<f32>) -> u32
 `
   )
-  .unimplemented();
+  .params(u => u.combine('inputSource', allInputSources))
+  .fn(async t => {
+    const makeCase = (x: number, y: number): Case => {
+      x = quantizeToF32(x);
+      y = quantizeToF32(y);
+      return { input: [vec2(f32(x), f32(y))], expected: u32(pack2x16snorm(x, y)) };
+    };
+
+    // Returns a value normalized to [-1, 1].
+    const normalize_f32 = (n: number): number => {
```

Comment:
odd mix of `snake_case` and `camelCase`

Reply:
Convert to camel, and cleaned up some other instances of this in this stack of PRs.